enterprise/app/api_keys/api_keys.tsx

```diff
@@ -140,7 +141,7 @@ export default class ApiKeysComponent extends React.Component<ApiKeysComponentPr
         request: new api_key.UpdateApiKeyRequest({
           id: apiKey.id,
           label: apiKey.label,
-          capability: apiKey.capability,
+          capability: apiKey.capability.slice(0),
```

Comment:
nit: slightly more idiomatic to do `[...apiKey.capability]`

Reply:
Done.